### PR TITLE
html editor

### LINKED ISSUE #197: React html code editor
As a user I would like to have code editor for html component

### PR DESCRIPTION
**Pull request by Chi-Chu319**
resolves #197

### CODE REVIEW

File: src/localization/en.json [hunk 1]
```diff
@@ -266,8 +267,11 @@
           "columnNumberType": "Number field",
           "submitButtonText": "Button text",
           "newFieldOption": "Option field",
-          "addCustomHtml": "Add HTML code in field",
           "contextVisibilityInfo": "Field management settings",
+          "html": {
+            "openEditor": "Open editor",
```
Comment (EVP1989) [inline src/localization/en.json:272]: I think this is not used anymore?

File: src/localization/fi.json [hunk 2]
```diff
@@ -265,8 +266,11 @@
           "columnNumberType": "Numerokenttä",
           "submitButtonText": "Painikkeen teksti",
           "newFieldOption": "Valintakenttä",
-          "addCustomHtml": "Lisää tähän HTML-sisältöä",
           "contextVisibilityInfo": "Kentän hallinnointi asetukset",
+          "html": {
+            "openEditor": "Avaa editori",
```
Comment (EVP1989) [inline src/localization/fi.json:271]: Same here, check localizations.

File: src/metaform-react/MetaformEditableHtmlComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      ...field,
+      html: updatedHtml
+    });
+  };
+
+  const modules = {
+    toolbar: {
+      container: [
+        ["bold", "italic", "underline", "strike", "blockquote"],
+        ["headingOne", "headingTwo", "headingThree"],
```
Comment (EVP1989) [inline src/metaform-react/MetaformEditableHtmlComponent.tsx:45]: Should this be heading choices?  Seems that it's not working properly.
Try this [{ 'header': [1, 2, 3, 4, 5, 6, false] }],

Also we could add text color property here also.

**Review by EVP1989 — COMMENTED** [review]
Added comments

File: src/metaform-react/MetaformEditableHtmlComponent.tsx (hunk 3 above)
Comment (villejuutila) [reply src/metaform-react/MetaformEditableHtmlComponent.tsx:45]: Was just testing it and I can confirm that works and correctly applies a dropdown menu for different header options.
```suggestion
        [{ "header": [ 1, 2, 3, 4, 5, 6, false ] }],
```

File: src/metaform-react/MetaformEditableHtmlComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    return null;
+  }
+
+  /**
+   * On field html update
+   *
+   * @param updatedHtml updated html
+   */
+  const onFieldHtmlUpdate = (updatedHtml: string) => {
+    console.log(updatedHtml);
```
Comment (villejuutila) [inline src/metaform-react/MetaformEditableHtmlComponent.tsx:34]: Remove console.log()

**Review by villejuutila — APPROVED** [review]
Approved with one comment.

**Review by EVP1989 — APPROVED** [review]
Approved

**Pull request merged**